src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -1032,10 +1032,8 @@ private void ProcessOutputRecord(ProcessOutputObject outputValue)
 
             if (outputValue.Stream == MinishellStream.Error)
             {
-                ErrorRecord record = outputValue.Data as ErrorRecord;
-                Dbg.Assert(record != null, "ProcessReader should ensure that data is ErrorRecord");
-                record.SetInvocationInfo(this.Command.MyInvocation);
-                this.commandRuntime._WriteErrorSkipAllowCheck(record, isNativeError: true);
+                // write stderr as string instead of as ErrorRecord
+                this.commandRuntime._WriteErrorSkipAllowCheck(outputValue.Data);
```

Comment:
`outputValue.Data` is still an `ErrorRecord`  - right?

Reply:
You're right, `outputValue.Data` is still an ErrorRecord.  I'll need to rework this.